MySensors.coffee

```diff
@@ -95,20 +95,18 @@ module.exports = (env) ->
           type: "number"
           unit: '%'
       }
-     
+
       @board.on("rfValue", (result) =>
         if result.sender is @config.nodeid
-          for sensorid in @config.sensorid
-            if result.sensor is sensorid
```

Comment:
consider a situation where one node with id x having multiple sensors like temp a, humidity b, motion c, distance d.  etc. each one requires to send different sensor data . how plugin code distinguish payload from each. let me know your  suggestion..